src/views/Receiving.vue

```diff
@@ -64,6 +64,7 @@ export default defineComponent({
         statusId: "PURCH_SHIP_SHIPPED"
       }
       await this.store.dispatch("shipment/findShipment", payload);
+      console.log(this.shipments)
```

Comment:
It seems we have missed a log

Reply:
Removed.
Thank you sir for reviewing my code.